docker/alpine/Dockerfile

```diff
@@ -17,8 +17,9 @@
 #   To run tests, use `--build-arg WITH_TESTS=true`
 #
 #   # Note: In this case, I would suggest to map port `5000` to `5000`. The user can always change the port according to their needs.
-#   docker run -it --rm --init --name agent -p5000:5000 \
-#     mtconnect/agent:2.0.0.12_RC18
+#   docker run -it --rm --init --name agent -p5000:5000 -v ./agent/agent.cfg:/mtconnect/config/agent.cfg \
+#        -v ./agent/Devices.xml:/mtconnect/config/Devices.xml \
+#        mtconnect/agent:2.0.0.12_RC18
```

Comment:
Either remove `2.0.0.12_RC18` or replace it with `latest` or update it on each release.

```suggestion
#        mtconnect/agent:latest
```

Reply:
removed version, latest is default